ouroboros-consensus/src/Ouroboros/Consensus/HardFork/Combinator/Embed/Binary.hs

```diff
@@ -124,6 +123,17 @@ protocolInfoBinary protocolInfo1 eraParams1 toPartialConsensusConfig1 toPartialL
     shape :: History.Shape '[blk1, blk2]
     shape = History.Shape $ exactlyTwo eraParams1 eraParams2
 
+
+blockForgingBinary :: forall blk1 blk2 m.
```

Comment:
Minor: style is to indent the whole type, as in `protocolInfoBinary` above.

Reply:
Fixed